x-pack/plugins/canvas/public/components/var_config/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import copy from 'copy-to-clipboard';
+import { VarConfig as MyComponent } from './var_config';
+import {
+  withKibana,
+  KibanaReactContextValue,
+  KibanaServices,
+} from '../../../../../../src/plugins/kibana_react/public';
+import { CanvasServices } from '../../services';
+
+import { ComponentStrings } from '../../../i18n';
+
+import { CanvasVariable } from '../../../types';
+
+const { VarConfig: strings } = ComponentStrings;
+
+interface Props {
+  kibana: KibanaReactContextValue<{ canvas: CanvasServices } & KibanaServices>;
```

Comment:
`public/index` has a `WithKibana` type definition that you can use here instead of building it yourself.  That might should be moved to a new location to make that more obvious.

Reply:
I tried using that other type definition and it was not having it...

```
Type '{ services: CoreStart & CanvasStartDeps & { canvas: CanvasServices; }; }' is not assignable to type 'KibanaReactContextValue<any>
```

Maybe I was miss using it? I was passing it like `FC<Props & WithKibanaProps>`